examples/advanced/sklearn-kmeans/jobs/sklearn_kmeans_base/app/custom/kmeans_assembler.py

```diff
@@ -33,7 +32,8 @@ def __init__(self):
         self.count = None
         self.n_cluster = 0
 
-    def get_model_params(self, data: dict):
+    def get_model_params(self, dxo: DXO):
+        data = dxo.data
         return {"center": data["center"], "count": data["count"]}
 
     def assemble(self, data: Dict[str, dict], fl_ctx: FLContext) -> dict:
```

Comment:
please update this method signature to

`def assemble(self, data: Dict[str, dict], fl_ctx: FLContext) -> DXO:`

for all child class and parent class in nvflare/app_common/aggregators/assembler.py

Thanks

Reply:
updated